packages/ember-htmlbars/tests/helpers/with_test.js

```diff
@@ -1,262 +1,13 @@
-    equal(view.$().text(), 'Señor Engineer: Tom Dale', 'precond - should be properly scoped after updating');
-
-    run(function() {
-      set(view, 'context.person.name', 'Yehuda Katz');
-    });
-
-    equal(view.$().text(), 'Señor Engineer: Yehuda Katz', 'should be properly scoped after updating');
-  });
-
-  QUnit.test('updating a property on the view should update the HTML', function() {
-    run(function() {
-      view.set('context.title', 'Señorette Engineer');
-    });
-
-    equal(view.$().text(), 'Señorette Engineer: Tom Dale', 'should be properly scoped after updating');
-  });
-}
-
-QUnit.module('Multiple Handlebars {{with foo as |bar|}} helpers', {
-  setup() {
-    Ember.lookup = lookup = { Ember: Ember };
-  },
-
-  teardown() {
-    runDestroy(view);
-
-    Ember.lookup = originalLookup;
-  }
-});
-
-QUnit.test('re-using the same variable with different #with blocks does not override each other', function() {
-  view = EmberView.create({
-    template: compile('Admin: {{#with admin as |person|}}{{person.name}}{{/with}} User: {{#with user as |person|}}{{person.name}}{{/with}}'),
-    context: {
-      admin: { name: 'Tom Dale' },
-      user: { name: 'Yehuda Katz' }
-    }
-  });
-
-  runAppend(view);
-  equal(view.$().text(), 'Admin: Tom Dale User: Yehuda Katz', 'should be properly scoped');
-});
-
-QUnit.test('should respect `isTruthy` field on a view', function() {
-  view = EmberView.create({
-    template: compile('{{#with view}}True{{else}}False{{/with}}'),
-    isTruthy: true
-  });
-  runAppend(view);
-
-  equal(view.$().text(), 'True');
-
-  run(function() {
-    set(view, 'isTruthy', false);
-  });
-
-  equal(view.$().text(), 'False');
-
-  run(function() {
-    set(view, 'isTruthy', true);
-  });
-
-  equal(view.$().text(), 'True');
-});
-
-QUnit.test('should respect `isTruthy` field on an object', function() {
-  view = EmberView.create({
-    template: compile('{{#with view.foo}}True{{else}}False{{/with}}'),
-    foo: {
-      isTruthy: true
-    }
-  });
-  runAppend(view);
-
-  equal(view.$().text(), 'True');
-
-  run(function() {
-    set(view, 'foo.isTruthy', false);
-  });
-
-  equal(view.$().text(), 'False');
-
-  run(function() {
-    set(view, 'foo.isTruthy', true);
-  });
-
-  equal(view.$().text(), 'True');
-});
-
-QUnit.test('should respect `isTruthy` field on the context object', function() {
-  view = EmberView.create({
-    template: compile('{{#with foo}}True{{else}}False{{/with}}'),
-    context: {
-      foo: {
-        isTruthy: true
-      }
-    }
-  });
-
-  runAppend(view);
-
-  equal(view.$().text(), 'True');
-
-  run(function() {
-    set(view, 'context.foo.isTruthy', false);
-  });
-
-  equal(view.$().text(), 'False');
-
-  run(function() {
-    set(view, 'context.foo.isTruthy', true);
-  });
-
-  equal(view.$().text(), 'True');
-});
-
-QUnit.test('the scoped variable is not available outside the {{with}} block.', function() {
```

Comment:
:ok_hand: "the scoped variable is not available outside the {{with}} block." (the first one)